cpp/betweenness_centrality_module/betweenness_centrality_online_module.cpp

```diff
@@ -0,0 +1,249 @@
+#include <algorithm>
+#include <thread>
+
+#include <mg_generate.hpp>
+#include <mg_graph.hpp>
+#include <mg_utils.hpp>
+
+#include "algorithm_online/betweenness_centrality_online.hpp"
+
+namespace {
+constexpr std::string_view kProcedureSet{"set"};
+constexpr std::string_view kProcedureGet{"get"};
+constexpr std::string_view kProcedureUpdate{"update"};
+constexpr std::string_view kProcedureReset{"reset"};
+
+constexpr std::string_view kArgumentCreatedVertices{"created_vertices"};
+constexpr std::string_view kArgumentCreatedEdges{"created_edges"};
+constexpr std::string_view kArgumentDeletedVertices{"deleted_vertices"};
+constexpr std::string_view kArgumentDeletedEdges{"deleted_edges"};
+constexpr std::string_view kArgumentNormalize{"normalize"};
+constexpr std::string_view kArgumentThreads{"threads"};
+
+constexpr std::string_view kFieldNode{"node"};
+constexpr std::string_view kFieldBC{"betweenness_centrality"};
+constexpr std::string_view kFieldMessage{"message"};
```

Comment:
Not sure if this is better than using `constexpr char const *` since we are working with C-API

Reply:
Changed :ok_hand: